lib/blocks.php

```diff
@@ -174,6 +174,13 @@ function gutenberg_render_block( $block ) {
 	 * @return string          Updated post content.
 	 */
 	function do_blocks( $content ) {
+		// If there are blocks in this content, we shouldn't run wpautop() on it later.
+		$priority = has_filter( 'the_content', 'wpautop' );
+		if ( false !== $priority && doing_filter( 'the_content' ) && has_blocks( $content ) ) {
+			remove_filter( 'the_content', 'wpautop', $priority );
+			add_filter( 'the_content', 'wpautop', $priority + 1 );
```

Comment:
Immediately re-adding wpautop here. 🤦‍♂️   Good example of why one shouldn't work during time set aside for recharging.